src/flamegraph/mod.rs

```diff
@@ -866,7 +873,9 @@ fn filled_rectangle<W: Write>(
             "y" => &buffer[y],
             "width" => &buffer[width],
             "height" => &buffer[height],
-            "fill" => &buffer[color]
+            "fill" => &buffer[color],
+            "x_samples" => &buffer[x_samples],
+            "width_samples" => &buffer[width_samples]
```

Comment:
I hate to recommend this, but given how many instances of this we may end up generating, I wonder if we should choose shorter labels for these. It could have a meaningful impact of file size for the SVG.

Separately, does SVG generally permit random attributes to be placed on XML elements like this? Judging by [this StackOverflow answer](https://stackoverflow.com/a/20265073/472927) I think we'll either want to add a simple flamegraph namespace (`fg`) or at least name these with the `data-` prefix.